CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -894,7 +894,6 @@ void CFLog1(CFLogLevel lev, CFStringRef message) {
     if (maxLength > sizeof(stack_buffer) / sizeof(stack_buffer[0])) {
         buffer = calloc(sizeof(char), maxLength);
         if (!buffer) {
-            -            __android_log_print(ANDROID_LOG_ERROR, tag, "Unable to allocate %d bytes for log message - truncating.", (int)axLength);
```

Comment:
tbh if the calloc fails bigger problems are going to happen down the road anyhow.

My opinion is just let it fail. It is not like this is a recoverable path or a container like Data